assets/css/base/_base.scss

```diff
@@ -1 +1,18 @@
 // Base styles.
+
+h1 {
+	font-size: 4em ;
+	font-weight: semi-bold ;
```

Comment:
`semi-bold` is not a valid property for `font-weight`. In all of these cases you should use the numeric values identified here: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Common_weight_name_mapping